Master/appstats.js

```diff
@@ -0,0 +1,164 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2017, Alert Logic, Inc
+ * @doc
+ * 
+ * The module for getting storage account settings.
+ * 
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+
+const async = require('async');
+const util = require('util');
+const moment = require('moment');
+
+const m_alUtil = require('../lib/al_util');
+
+const STATS_PERIOD_MINUTES = 15;
+const APP_FUNCTIONS = ['Master', 'O365WebHook', 'Updater'];
+
+var azureStorage = require('azure-storage');
+var TableQuery = azureStorage.TableQuery;
+var TableUtilities = azureStorage.TableUtilities;
+var TableService = null;
+
+// For "AccountName=teststorageaccount" returns 
+// {
+//      "AccountName": "teststorageaccount"
+// }
+var assignmentToObj = function(assignString) {
+    const splitIndex = assignString.indexOf('=');
+    const key = assignString.slice(0, splitIndex);
+    const value = assignString.slice(splitIndex + 1);
+    var obj = {};
+    
+    obj[key] = value;
+    return obj;
+};
+
+// For "AccountName=teststorageaccount;AccountKey=key" returns 
+// {
+//      "AccountName": "teststorageaccount",
+//      "AccountKey": "key"
+// }
+var assignStringToJson = function(assignString) {
+    const params = assignString.split(';');
+    return params.reduce(function(acc, current) {
+            return Object.assign(acc, assignmentToObj(current));
+        },{});
+};
```

Comment:
Rather use https://www.npmjs.com/package/parse-key-value than the above two functions.